### DISCUSSION TITLE
select and edit multiple objects

### DISCUSSION THREAD

**Discussion by wiesehahn** [streetcomplete/StreetComplete#5922] [Ideas]
probably there are already issues/discussions on this topic but I could not find them so far. 

Let's say I would like to edit multiple map objects of the same type in the same way. E.g. answer if a tree has needles or leaves. Currently this is quite tedious as we have to to this for one tree after the other. 
In certain cases it would be much faster if there was an option to select multiple objects or all objects in viewport and then answer the question for all of them at once. (Possibly in this case the should be an extra warning/info but it would still be much more efficient for many tasks).

**Comment by riQQ** [comment]
See section *'Answer the same quest for multiple elements at once'* in the discussion https://github.com/streetcomplete/StreetComplete/discussions/2789#discussioncomment-646707.

**Reply by mnalis** [reply]
Or, if **_all_ are the same** in that place, and there is many of them, you can leave a Note (by selecting `Uh...`  / `Can't say` / `Leave a note`, or by long pressing and the screen and choosing `Create new note`) and then doing it later in comfort of desktop editor like [JOSM](https://wiki.openstreetmap.org/wiki/JOSM) which allows such multiselect changes to be done easily.

Or, if you want to do it on the ground, you can long press on the screen and choose `Open location in another app` and choose [Vespucci](https://wiki.openstreetmap.org/wiki/Vespucci) mobile editor which has support for multiselect. (but as noted in those linked issues, multiselect on mobile touch screens will be significantly less comfortable then one on desktop)
